tests.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+import flask
+import os
+import tempfile
+import urllib.request
+
+SITE_LOCATION = "http://localhost:5000"
```

Comment:
Notice that you implemented flake8 in the CI, but this code won't pass the flake8 test, and so next CI will probably fail.

Reply:
@flippy1989 ??

Reply:
I addressed this matter in another commit, all the code was pep8 tested and will pass the flake8